### ISSUE TITLE
[Feature Request]: Guest role

### ISSUE THREAD

**Issue by PovilasID** [sinamics/ztnet#257]
### 🚀 Feature Summary

A read only user type that is only able to check their ID and network state

### 📝 Detailed Description

It would be nice to have a user that is not able to create networks, they just get a list of networks that they are a member for ids for them and bit of documentation like links to clients how to download stuff.

### 🎯 Use Case

If users lose/can not find an email with IDs and instructions how to join the network or think the network is down or what devices are connected it would be nice to have a place where they can verify up to date information. 

### 💡 Willing to Contribute

None

**Comment by sinamics** [comment]
Interesting
It sounds like we're talking about a status page where you just enter your node ID and it shows you all the networks you're part of, plus their status.

This wouldn't necessarily need a full user account, which keeps it simple. But yeah, I'll have to think of a smart way to do this that's safe and ensuring that it doesn't compromise any security aspects.

**Comment by PovilasID** [comment]
You are correct, user account is not necessary! I am little worried that users may not know where the device ID is but that can be explained on the page and list of clients and how to install them definitely does not need a login.
Network status could be chosen to be almost publicly displayed.
However device status would have to be an option in network settings "Permit device status validation" or something to that effect, per network or per device... This one is more for validating that you successfully added that smartphone to people who are already inside of a secured network.
